docs/kiali/03-01-kiali.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Access Kiali
+type: Details
+---
+
+Kiali can be easily accessed from the left menu on the [Kyma Console](/components/console/#overview-overview) by clicking on the `Service Mesh` item.
+Once authenticated, the main Kiali dashboard will show a summary of the service mesh status and the left side menu offers different features such as graphs or configuration validation:
+![Kiali menu item](assets/overview.png)
```

Comment:
I'm still not convinced to bundle here screenshots as they will outdate so fast.. but if @bszwarc wants them I'm fine :)